src/main/java/software/coley/llzip/format/write/CopyZipWriterStrategy.java

```diff
@@ -0,0 +1,134 @@
+			writeIntLE(os, ZipPatterns.LOCAL_FILE_HEADER_QUAD);
+			// Minimum version
+			writeShortLE(os, fileHeader.getVersionNeededToExtract());
+			// General purpose bit flag
+			writeShortLE(os, fileHeader.getGeneralPurposeBitFlag());
+			// Compression method
+			writeShortLE(os, fileHeader.getCompressionMethod());
+			// Last modification time
+			writeShortLE(os, fileHeader.getLastModFileTime());
+			// Last modification date
+			writeShortLE(os, fileHeader.getLastModFileDate());
+			// CRC32
+			writeIntLE(os, fileHeader.getCrc32());
+			// Compressed size
+			writeIntLE(os, (int) fileHeader.getCompressedSize());
+			// Uncompressed size
+			writeIntLE(os, (int) fileHeader.getUncompressedSize());
+			// File name length
+			writeShortLE(os, fileHeader.getFileNameLength());
+			// Extra field length
+			writeShortLE(os, fileHeader.getExtraFieldLength());
+			// File name
+			os.write(ByteDataUtil.toByteArray(fileHeader.getFileName()));
+			// Extra field
+			os.write(ByteDataUtil.toByteArray(fileHeader.getExtraField()));
+			// Compressed data
+			os.write(ByteDataUtil.toByteArray(fileHeader.getFileData()), 0, (int) fileHeader.getCompressedSize());
+		}
+
+		// Write central directory file headers.
+		for (final CentralDirectoryFileHeader directory : archive.getCentralDirectories()) {
+			// Header
+			writeIntLE(os, ZipPatterns.CENTRAL_DIRECTORY_FILE_HEADER_QUAD);
+			// Made by
+			writeShortLE(os, directory.getVersionMadeBy());
+			// Minimum version
+			writeShortLE(os, directory.getVersionNeededToExtract());
+			// General purpose bit flag
+			writeShortLE(os, directory.getGeneralPurposeBitFlag());
+			// Compression method
+			writeShortLE(os, directory.getCompressionMethod());
+			// Last modification time
+			writeShortLE(os, directory.getLastModFileTime());
+			// Last modification date
+			writeShortLE(os, directory.getLastModFileDate());
+			// CRC32
+			writeIntLE(os, directory.getCrc32());
+			// Compressed size
+			writeIntLE(os, (int) directory.getCompressedSize());
+			// Uncompressed size
+			writeIntLE(os, (int) directory.getUncompressedSize());
+			// File name length
+			writeShortLE(os, directory.getFileNameLength());
+			// Extra field length
+			writeShortLE(os, directory.getExtraFieldLength());
+			// File comment length
+			writeShortLE(os, directory.getFileCommentLength());
+			// Disk number where file starts
+			writeShortLE(os, directory.getDiskNumberStart());
+			// Internal file attributes
+			writeShortLE(os, directory.getInternalFileAttributes());
+			// External file attributes
+			writeIntLE(os, directory.getExternalFileAttributes());
+			// Relative offset of local file header
+			writeIntLE(os, (int) directory.getRelativeOffsetOfLocalHeader());
+			// File name
+			os.write(ByteDataUtil.toByteArray(directory.getFileName()));
+			// Extra field
+			os.write(ByteDataUtil.toByteArray(directory.getExtraField()));
+			// File comment
+			os.write(ByteDataUtil.toByteArray(directory.getFileComment()));
```

Comment:
I don't think these comments are necessary since they mirror the getter names on a near 1-to-1 mapping